src/bid_order.rs

```diff
@@ -91,16 +93,7 @@ impl BidOrderV2 {
 
     /// Returns the remaining amount of base in the order
     pub fn get_remaining_base(&self) -> Uint128 {
-        self.base.amount
-            - self
-                .events
-                .iter()
-                .map(|event| match &event.action {
-                    Action::Fill { base, .. } => base.amount,
-                    Action::Reject { base, .. } => base.amount,
-                    _ => Uint128::zero(),
-                })
-                .sum::<Uint128>()
+        self.base.amount - self.remaining_base
```

Comment:
would this just be `self.remaining_base`?

Reply:
it's `self.base.amount - self.accumulated_base` now